jobs/mprsyncup

```diff
@@ -175,7 +175,7 @@ if [ "${PORTS_CHANGED}" -eq 1 ]; then
     # Intentionally split PLATFORMS on whitespace.
     for PLATFORM in $PLATFORMS; do
         INDEX="PortIndex_darwin_${PLATFORM}"
-        ${PORTINDEX} -p "macosx_${PLATFORM}" -o "${INDEX}" | ${SED} "s/^/Updating ${INDEX}:"$'\t'"/" | expand -t 40,48,56,64,72,80 &
+        ${PORTINDEX} -p "macosx_${PLATFORM}" -o "${INDEX}" | ${SED} "s/^/Updating ${INDEX}:	/" | expand -t 40,48,56,64,72,80 &
```

Comment:
I agree that the literal tab is fragile. `printf` had slipped my mind; I'll use that instead.